cmd/rm-main.go

```diff
@@ -35,6 +35,18 @@ import (
 // rm specific flags.
 var (
 	rmFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "versions",
+			Usage: "remove objects versions as well",
+		},
+		cli.StringFlag{
+			Name:  "rewind",
+			Usage: "back in time before removal",
```

Comment:
```suggestion
			Usage: "revert to older version of object(s) in time",
```

Reply:
> Usage: "revert to older version of object(s) in time",

@harshavardhana the thing is that if --rewind is used with --versions, it will remove all versions prior to the specified rewind argument. So I think this comment can be misleading

Reply:
@vadmeste revert means removal as well - it implies overwrite.